robot/tests/.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
don't use -a if you don't know what's being committed

Reply:
I thought I did know what I had committed, but I didn't see anything about that. What is it?

Reply:
It's just some junk files that `pytest` uses. I just opened #27; add it to the `.gitignore` file and redo this commit.